system/jlib/jstring.cpp

```diff
@@ -2761,3 +2761,34 @@ StringBuffer& StringBuffer::operator=(StringBuffer&& value)
     swapWith(value);
     return *this;
 }
+
+
+bool loadBinaryFile(StringBuffer & contents, const char *filename, bool throwOnError)
```

Comment:
given it's named loadBinaryFile, I might expect it to load into a MemoryBuffer,
or should this be called loadTextFile, given it's populating a null terminated buffer ?

Reply:
I went down the same path.  The problem with the MemoryBuffer is that I wanted it to be \0 terminated, and that would have caused a reallocation for a memory buffer.  If it was called loadTextFile it would imply translating \r\n to \n.  This was my best option.

Reply:
fair enough.